src/interface/sparse_pack_base.cpp

```diff
@@ -163,7 +163,7 @@ SparsePackBase SparsePackBase::Build(T *pmd, const PackDescriptor &desc,
     leading_dim += 2;
   }
   pack.pack_ = pack_t("data_ptr", leading_dim, pack.nblocks_, max_size);
-  auto pack_h = Kokkos::create_mirror_view(pack.pack_);
+  pack.pack_h_ = Kokkos::create_mirror_view(pack.pack_);
```

Comment:
The implication is that now the host mirror contains device views. This is intentional, but also not a bug fix. Previously the host mirror was not persistent in the pack. It's now persistent, which enables us to access, e.g., the `label` method for the packed views, which isn't available otherwise as `pack_` points to device memory.